runestone/shortanswer/js/shortanswer.js

```diff
@@ -134,7 +134,7 @@ ShortAnswer.prototype.submitJournal = function () {
 
 
     this.setLocalStorage({answer: value, timestamp: new Date()})
-    this.logBookEvent({'event': 'shortanswer', 'act': JSON.stringify(value), 'div_id': this.divid});
+    this.logBookEvent({'event': 'shortanswer', 'act': value, 'div_id': this.divid});
```

Comment:
Why remove JSON.stringify ?

Reply:
Because the server doesn't JSON decode it. Hence, there's an extra level of JSON introduced, which this commit fixes.

It doesn't need to be JSON encoded then decoded, since logBookEvent already does this.